source/bayes_simulation.Rmd

```diff
@@ -45,112 +45,171 @@ correctly identifies 80 percent as "OK" but says that 20 percent are
 
-`           IF a between 28 86         `
+At our mechanic's yard, we ask "how many red cars here are broken"?  To calculate that, we first count the number of red cars, then the number of those red cars that are broken, then calculate the ratio: `red_cars_broken / red_cars`.
 
-If it's between "28" and "86" (those that say "good")
+We could just as well have worked in percentages, and we would have gotten the same answer: `percentage_of_red_cars_broken / percentage_of_cars_that_are_red`, since
+that is `(red_cars_broken / 100) / (red_cars / 100)`.  And probabilities would have worked too, given our assumption that the counts are good representations of the yard: `P(red_cars_broken_in_the_yard) / P(red_cars_in_the_yard)`.
 
-`           SCORE a z         `
+**Probability relationships: conditional probability**
 
-Score this number
+We return to the quantity computed above: the **number** of cars being faulty *given* that the mechanic said they were OK.  We use this ratio as an estimate of the probability that that will be true, i.e: the **probability** of cars being faulty *given* that the mechanic said they were OK.
 
-`           END         `
+This is called a *conditional probability*, $P(X | Y)$.  Given a certain outcome Y, calculate the probability of X occurring.
 
-End the IF condition
+Conditional probabilities are calculated as follows:
 
-`           END         `
+$P(X | Y) = P(X \cup Y) / P(Y)$
 
-End REPEAT loop
+From our car example, what is the probability of a car being broken given that a mechanic said it was OK?  It is, out of all the cars that the mechanic said were OK, the fraction that weren't or:
 
-`           COUNT z between 28 30 k         `
+$P(car_broken & mechanic_said_OK) / P(mechanic_said_OK)$.
 
-How many of the SCORED numbers were between "28\--30" (faulty, "OK")
+**Example: conditional probability**
 
-`           SIZE z s         `
+Let's discuss a very relevant example.  You get a COVID test, and the test is negative.  Now, you would like to know what the chance is of you having COVID.
 
-How many numbers were scored
+We have the following information:
 
-`           DIVIDE k s kk         `
+- 1.5% of people in your area have COVID
+- The false positive rate of the tests (i.e., that they detect COVID when it is absent) is very low at 0.5%
+- The false negative rate (i.e., that they fail to detect COVID when it is present) is quite high at 40%
 
-What proportion were faulty, "OK"
+You would like to know the probability of having COVID given that your test was negative.  Using the conditional probability relationship from above, we can write:
 
-`           PRINT kk         `
+$P(covid | test_negative) = P(covid \cup test_negative) / P(test_negative)$
 
-Print result
+We notice that $P(covid \cup test_negative) = P(test_negative | covid) * P(covid) = .4 * .015 = 0.006$.
+We also observe that $P(test_negative) = P(test_negative | covid) + P(test_negative | not_covid) = 0.4 + 0.995 = 1.395$.
```

Comment:
There's two ways of ending up with a negative test - as there were two ways of ending up with the mechanic saying the car was OK in the example above.  You could end up with negative test when you do have it (the thing we just calculated) or you could end up with a negative test when you don't have it.

Reply:
Careful - the two probabilities must add up to something less than one - right?

There are the two ways, one of which is P(T- ^ C+) (that you have).  Notice this is ^ not | - the probablility that you BOTH have a negative test AND that you have COVID.

The other way is P(T- ^ C-) - negative test AND no COVID.  Add probability of these these two mutually exclusive events together to get the probably of EITHER of these outcomes happening.

And this is another probability rule that we haven't discussed here (and do earlier in the book) that the probability of EITHER of two exclusive events happening, say A and B - is P(A) + P(B).

Reply:
WOOPS!!